src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -953,20 +953,6 @@ public static String onClientSendMessage(String message)
         return MinecraftForge.EVENT_BUS.post(event) ? "" : event.getMessage();
     }
 
-    /**
-     * Mimics the behavior of {@link net.minecraft.client.renderer.ItemBlockRenderTypes#getRenderType(BlockState, boolean)}
-     * for the input {@link RenderType}.
-     */
-    @NotNull
-    public static RenderType getEntityRenderType(RenderType chunkRenderType, boolean fabulous)
```

Comment:
Should we avoid this breaking change by reinstanting this method, deprecating this method for removal in the next major version/breaking changes window, and redirecting it to `RenderTypeHelper#getEntityRenderType`?

Reply:
I wouldn't say it's a breaking change since it's in `ForgeHooksClient` and the whole class is marked as internal, but you can have the final say on it.

Reply:
Keep and forward.